cpp/src/c_api/k_truss.cpp

```diff
@@ -14,19 +14,19 @@
  * limitations under the License.
  */
 
-#include "c_api/abstract_functor.hpp"
-#include "c_api/graph.hpp"
-#include "c_api/induced_subgraph_result.hpp"
-#include "c_api/resource_handle.hpp"
-#include "c_api/utils.hpp"
-
 #include <cugraph_c/algorithms.h>
 
 #include <cugraph/algorithms.hpp>
 #include <cugraph/detail/shuffle_wrappers.hpp>
 #include <cugraph/detail/utility_wrappers.hpp>
 #include <cugraph/graph_functions.hpp>
 
+#include <c_api/abstract_functor.hpp>
```

Comment:
These should be using the "" syntax as before.  You might need to update your pre-commit utilities.

Reply:
Got it. I didn't catch that. Thanks